content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -94,9 +94,12 @@ Follow our instructions on how to
 [use Helm and Tiller to manage the Istio deployment](/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install).
 
 ## Verifying the installation
+{{< tip >}}
+Note that istio-egressgateway is not present by default if you used Helm to install.
```

Comment:
Do not add "note" to a callout, it's redundant. The text doesn't provide enough context. Tips work best if you place the reason for the tip at the beginning. 
```suggestion
If you installed Istio using Helm, the `istio-egressgateway` service is not present by default. Consider this fact as you verify that the services are deployed.
```